cumulusci/cumulusci.yml

```diff
@@ -1,7 +1,14 @@
 cumulusci:
     keychain: cumulusci.core.keychain.EncryptedFileProjectKeychain
-
+    
 tasks:
+    activate_flow:
+        group: Metadata Transformations
+        description: Activates Flows identified by a given list of Developer Names
+        class_path: cumulusci.tasks.salesforce.ActivateFlowProcesses.ActivateFlowProcesses
```

Comment:
Python modules should be named with snake_case and should not have the same name as a class they contain. (There are some counterexamples to this in the existing codebase, but we want to fix them and definitely don't want to add more.)

Please rename the module to activate_flow.py and the class to ActivateFlow